src/index.html

```diff
@@ -22,6 +22,347 @@
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav_item">
+            <a
+              class="nav_link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="cards__container">
+        <div
+          class="product__card"
+          data-qa="card"
+        >
+          <div class="product__image"></div>
+
+          <div class="product__info">
+            <div class="product__name">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </div>
+
+            <div class="product__code">Product code: 195434</div>
+
+            <div class="product__review-box">
+              <div class="product__stars">
+                <div class="product__star product__star--highlighted"></div>
+                <div class="product__star product__star--highlighted"></div>
+                <div class="product__star product__star--highlighted"></div>
+                <div class="product__star product__star--highlighted"></div>
+                <div class="product__star"></div>
+              </div>
+
+              <div class="product__review-counter">Reviews: 5</div>
+            </div>
+
+            <div class="product__price-box">
+              <div class="product__price-label">Price:</div>
+              <div class="product__price-value">$2,199</div>
+            </div>
+
+            <a href="#"
+              data-qa="card-hover"
+              class="buy-button"
+            >
+              buy
+            </a>
+          </div>
+        </div>
+        <div class="product__card">
```

Comment:
Repeat the kebab-case corrections for class names in all `product__card` elements as previously mentioned.